src/System.Diagnostics.FileVersionInfo/tests/NativeConsoleApp/NativeConsoleApp.cpp

```diff
@@ -0,0 +1,8 @@
+#include <stdio.h>
+
+int main()
+{
+    printf("Hello World");
```

Comment:
Nit: We never actually run the application, right?  Just wondering if you even need this printf, or if the whole file could just be:
```C++
int main() { return 0; }
```